### PR TITLE
#151715868 user avatar selection on signup

### PR DESCRIPTION
**Pull request by gbengaoyetade**
#### What does this PR do?
Add avatar selection to the signup page
#### Description of Task to be completed?
Users, once they visit the signup page, should be able to select an avatar before signup is successful.
#### How should this be manually tested?
Visit the application signup page 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#151715868
#### Screenshots (if appropriate)
avatar-signup

### CODE REVIEW

File: public/js/controllers/index.js [hunk 1]
```diff
@@ -24,6 +26,7 @@ angular.module('mean.system')
         $scope.error = '';
        $http.post('/api/auth/login', userInput)
        .success((response) => {
+         console.log(response);
```
Comment (segunolalive) [inline public/js/controllers/index.js]: Please remove this
screen shot 2017-10-06 at 3 06 33 pm

The user data is in the console

**Comment by segunolalive** [comment]
Good job.

Keep up the good work

File: public/js/controllers/index.js [hunk 2]
```diff
@@ -24,6 +26,7 @@ angular.module('mean.system')
...
         if(response.token) {
           window.localStorage.setItem('cfhToken', response.token);
```
Comment (dinobi) [inline public/js/controllers/index.js]: use the angular window service

**Review by segunolalive — APPROVED** [review]
Great job

File: public/js/controllers/index.js (hunk 2 above)
Comment (gbengaoyetade) [reply public/js/controllers/index.js]: I've changed it to just localStorage

**Review by dinobi — APPROVED** [review]

**Pull request merged**